lib/src/core/utils/dimensions_constants.dart

```diff
@@ -2,4 +2,49 @@ abstract class Dimensions {
   static const int splashScreenDuration = 4500;
```

Comment:
Review this file, try to keep here only the constants that are used in more than one place, and leave the specific const inside each file that uses it

Reply:
Done